actors/runtime/src/main/java/cloud/orbit/actors/Stage.java

```diff
@@ -715,7 +715,7 @@ public void setDeactivationTimeout(long deactivationTimeoutMs)
                 .findFirst()
                 .orElseGet(() ->
                 {
-                    final ResponseCaching responseCaching = new ResponseCaching();
+                    final DefaultResponseCachingExtension responseCaching = new DefaultResponseCachingExtension();
```

Comment:
Syntax sugar... make the responseCaching of type ResponseCachingExtension (coding to interfaces)?